### DISCUSSION TITLE
How to access the matrix/ dataframe that generated the PCA & volcano plot from FragPipeAnalyst?

### DISCUSSION THREAD

**Discussion by MelissaVorster** [Nesvilab/FragPipe-Analyst-doc#34] [Q&A] [answered]
Good day Team
I would like to see what data is being used to generate the PCA plot. Which of the "Choose a dataset to save" options include this matrix? Or is there another way that I can access this matrix? 

On that note, is it possible to download the data structure used to create the volcano plot as well?

Kind regards
Melissa

**Comment by hsiaoyi0504** [comment, marked as answer]
The data of PCA plot wasn't included in the current download option, but we will consider it. For the volcano plot, it's underlying data is available as "DE result".

**Comment by hsiaoyi0504** [comment]
Closed due to no response.
